charts/jenkins-jobs/templates/_helpers.tpl

```diff
@@ -342,7 +342,8 @@ configNode << 'com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPri
   {{- end }}
   usernameSecret({{ .usernameSecret | default false}})
   privateKeySource(class:"com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey\$DirectEntryPrivateKeySource") {
-    privateKey('{{ .privateKey }}')
+    privateKey('''{{ .privateKey }}''')
```

Comment:
Can't `quote`  or `squote` be usefull here?

Reply:
If it handles triple quote yes 😅(i don’t like quotes though. Not clear enough)